docs/src/practices/tracking_issues.md

```diff
@@ -0,0 +1,55 @@
+# Tracking issues
+
+`nearcore` uses tracking issues on github to make it easier to follow progress
+on individual projects.
+
+## Purpose
+
+Different types of audiences want to track progress on feature development on `nearcore`:
+- Other engineers contributing to the feature development but who may have
+  missed some progress while they were unavailable.
+- Pagoda management who would like to have an overview of which teams are
+  working on what features and how much work is left to be done.
+- The general NEAR community who are interested in development of new features
+  in the protocol.
+
+Using tracking issues can help the above audience come up to speed with projects
+quickly and makes it easy for engineers working on projects to share updates.
+
+## Background
+
+The idea of tracking issues is also used to track project work in the Rust
+language.  See [this
+post](https://internals.rust-lang.org/t/how-the-rust-issue-tracker-works/3951)
+for a rough description and
+[these](https://github.com/rust-lang/rust/issues/101840)
+[issues](https://github.com/rust-lang/rust/issues/100717) for how they are used
+in Rust.
+
+## Guidelines
+
+- There should be one tracking issue per project.  Discretion can be used to
+  decide what kind of project necessitates a tracking issue.
+- Tracking issues should be maintained on the `nearcore` github repository.  If
```

Comment:
An important bullet point here: 

* keep tracking issues up-to date. At minimum, add all new context as a comment, but preferably edit the origianal description of the issue to reflect the current status.